test/runtests.jl

```diff
@@ -9,7 +9,14 @@ end
 
 using Documenter
 DocMeta.setdocmeta!(ImageCore, :DocTestSetup, :(using ImageCore); recursive=true)
+
+# ReferenceTests uses ImageInTerminal as a default image rendering backend, we need to
+# temporarily disable it when we do doctest.
+# TODO: ReferenceTests doesn't yet support this switch. That's why we need ImageInTerminal dependency.
+using ImageInTerminal
+ImageInTerminal.disable_encoding()
 doctest(ImageCore, manual = false)
+ImageInTerminal.enable_encoding()
```

Comment:
The idea and more elegant way I have in mind to do this is:

```julia
ReferenceTests.disable_encoding(:all) do
    doctest(ImageCore, manual = false)
end
```

But ReferenceTests is a little bit broken and fixing it requires more effort than I can put now.

@timholy we could merge this version and clean up the _direct_ test dependency of ImageInTerminal in the future. (ReferenceTests still requires ImageInTerminal)